boards/v2/synopsys/emsdp/Kconfig.emsdp

```diff
@@ -4,12 +4,46 @@
 # SPDX-License-Identifier: Apache-2.0
 
 config BOARD_EMSDP
-	bool "EM Software Development Platform"
-	depends on SOC_ARC_EMSDP
+	select SOC_ARC_EMSDP
 	help
 	  The ARC EM Software Development Platform (emsdp) is an FPGA based
 	  development platform intended to support ARC licenses in developing
 	  their software for the ARC EM processor family and ARC EM Subsystems.
 	  It has the support for ARC EM4, EM5D, EM6, EM7D, EM9D and EM11D
 	  processors. ARC EM Enhanced Security Package (ESP) and ARC EM
 	  Subsystems (DFSS, SCSS, DSS) are also supported.
+
+config BOARD_EMSDP_EMSDP_EM4
+	bool
```

Comment:
Please remove redundant `bool` in this file.

```suggestion
```

Reply:
Fixed